exchanges/orderbook/orderbook_test.go

```diff
@@ -588,37 +564,34 @@ func deployUnorderedSlice() []Item {
 
 func TestSorting(t *testing.T) {
 	var b Base
+	b.VerifyOrderbook = true
 
 	b.Asks = deployUnorderedSlice()
 	err := b.Verify()
-	if err == nil {
-		t.Fatal("error cannot be nil")
+	if !errors.Is(err, errPriceOutOfOrder) {
+		t.Fatalf("error expected %v received %v", errPriceOutOfOrder, err)
 	}
 
-	SortAsks(nil)
-
-	SortAsks(b.Asks)
+	b.Asks.SortAsks()
 	err = b.Verify()
 	if err != nil {
 		t.Fatal(err)
 	}
 
 	b.Bids = deployUnorderedSlice()
 	err = b.Verify()
-	if err == nil {
-		t.Fatal("error cannot be nil")
+	if !errors.Is(err, errPriceOutOfOrder) {
+		t.Fatalf("error expected %v received %v", errPriceOutOfOrder, err)
 	}
 
-	SortBids(nil)
-
-	SortBids(b.Bids)
+	b.Bids.SortBids()
 	err = b.Verify()
 	if err != nil {
 		t.Fatal(err)
 	}
 }
 
-func deploySliceOrdered() []Item {
+func deploySliceOrdered() Items {
```

Comment:
This is a really useful function and theres a good chance this, along with deploySliceUnordered, might get used by someone else in the future. However the next person is restricted to a 1000 length slice. If the length is modified by someone in the future, your current tests may start failing. So for that reason could we add in a length param for these two functions

Reply:
When somebody needs to change their slice length they can update everything to accommodate.